### PR TITLE
feature/zcs-1963 - additional zimbra-imapd documentation

### PR DESCRIPTION
**Pull request by gordyt**
This PR contains additional documentation related to `zimbra-imapd` configuration and migration.  Have created an HTML export of the entire admin guide [here](https://zimbra.github.io/adminguide/imapd/) to facilitate review. Just look for *Zimbra IMAPD Server* in the navigation bar on the left.  Everything is under that main heading.

### CODE REVIEW

File: imap_upstream.adoc [hunk 1]
```diff
@@ -31,4 +31,156 @@ The available balancing algorithms are as follows:
...
+* `imapd.pid`
+
+== Other Files
+
+Activity statistics are collected in `/opt/zimbra/zmstat/imapd.csv`.
+
+
+= Migrating to an imapd pool
+
+When migrating to dedicated pool of IMAP servers, several steps must
```
Comment (tmclane) [inline imap_upstream.adoc]: migrating to => migrating to a

File: imap_upstream.adoc [hunk 2]
```diff
@@ -31,4 +31,156 @@ The available balancing algorithms are as follows:
...
+appropriately to accommodate expected IMAP traffic levels. It is
+recommended that you test each server to ensure that it is functioning
+correctly; this can be done via command line with openssl, or by
+configuring an existing IMAP client to point directly at the
+server. These servers should not be listed in the
+`zimbraReverseProxyUpstreamImapServers` attribute yet.
+2. (optional) Specify a load-balancing algorithm using the
+`zimbraImapLoadBalancingAlgorithm` attribute. The default is
+`ClientIPHash`, but custom load-balancing algorithms can be specified as
+well.
```
Comment (tmclane) [inline imap_upstream.adoc]: Maybe we should note that no other algorithms are currently available?

File: index.adoc [hunk 3]
```diff
@@ -114,10 +120,6 @@ include::cmdlineutils.adoc[]
 
 <<<
 
-include::remote_imap.adoc[]
-
-<<<
-
 include::spnego.adoc[]
```
Comment (tmclane) [inline index.adoc]: Thanks for fixing the location!

File: imap_upstream.adoc (hunk 1 above)
Comment (gordyt) [inline imap_upstream.adoc]: Thank you!  This has been corrected in a fix-up commit.  I did not regenerate the HTML for the `gh-pages` view yet.

File: imap_upstream.adoc (hunk 2 above)
Comment (gordyt) [inline imap_upstream.adoc]: Agreed!  Have updated the wording in a fix-up commit.  I did not regenerate the HTML for the `gh-pages` view yet.

**Pull request merged**